tools/benchmark/cmd/watch.go

```diff
@@ -137,8 +137,8 @@ func watchFunc(cmd *cobra.Command, args []string) {
 	<-pdoneC
 
 	// put phase
-	// total number of puts * number of watchers on each key
-	eventsTotal = watchPutTotal * (watchTotal / watchedKeyTotal)
+	// total number of puts * (number of watchers on each key) / (number of events triggered per put)
+	eventsTotal = watchPutTotal * (watchTotal / watchedKeyTotal) / (watchedKeyTotal / watchKeySpaceSize)
```

Comment:
@heyitsanthony Sorry, I am still confused.
Now it's not clear to me what `eventsTotal` actually means to this benchmark testing.

So we want to trigger watch events for `eventsTotal` times.  Then don't we just need correctly shut down the watch operation when it reaches the eventsTotal like

```
atomic.LoadInt32(&nrRecvCompleted) >= int32(eventsTotal)
```

I am not sure what's the best way to get the eventsTotal with these multiple dimensions.